<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveViewPccupancyDataWidget</class>
 <widget class="QWidget" name="LiveViewPccupancyDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelIcon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resource/resource.qrc">:/liveview_occupancy/liveview_occupancy/In_1.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelText">
     <property name="styleSheet">
      <string notr="true">QLabel
{
	padding-left: 15px;
	color: #FFFFFF;
	font: 48pt;
}</string>
     </property>
     <property name="text">
      <string>In:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: #FFFFFF;
	font: 48pt;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
